tests/e2escenarios/helper/helper_oc.go

```diff
@@ -106,3 +106,19 @@ func (oc *OcRunner) GetComponentDC(component string, app string, project string)
 
 	return string(session.Wait().Out.Contents())
 }
+
+// AfterFailed runs certain oc commands like "oc get pod",
+// "oc get dc" and "oc get dc -o yaml" to analyze failure cause
+func (oc *OcRunner) AfterFailed() {
+	getPod := CmdRunner(oc.path, "get", "pod")
```

Comment:
isn't this missing `if`statement to make sure that this gets printed only on error?

Reply:
Yup, It is a miss.

Fixed